src/test/java/au/org/ala/names/search/ALANameSearcherTest.java

```diff
@@ -524,7 +525,7 @@ public void testPhraseMatch4() {
             NameSearchResult nsr = null;
             nsr = searcher.searchForRecord(name, null);
             assertNotNull(nsr);
-            assertEquals("http://id.biodiversity.org.au/name/apni/190511", nsr.getLsid());
```

Comment:
This original line is what Travis is reporting as the actual result, and failing the test currently.